src/components/MapInput/index.js

```diff
@@ -115,10 +116,18 @@ const MapInput = ({ className, value, onChange, mapOptions, ...otherProps }) =>
   }, [marker, location, hasLocation]);
 
   useEffect(() => {
-    if (!value?.geometrie?.coordinates && !value?.addressText) return;
+    if (!map) return;
+    if (!value?.location && !value?.addressText) return;
+
+    if (value?.location?.lat > 0) {
+      if (map && !isEqual(value?.location, state.location)) {
```

Comment:
The check for `map` is redundant; this code cannot be reached whenever `map` is falsy (see line 119).